src/applications/gi-sandbox/components/CautionFlagAdditionalInfo.jsx

```diff
@@ -18,29 +18,31 @@ export const CautionFlagAdditionalInfo = ({
       : `This school has ${validFlags.length} cautionary warnings`;
   return (
     <div className="usa-alert usa-alert-warning vads-u-background-color--white vads-u-padding-x--0p5 vads-u-padding-y--2">
-      <div
-        className="usa-alert-body "
-        onClick={() => toggleExpansion(!expanded)}
-      >
-        <div className="vads-u-display--flex">
-          <div>
-            <h4 className="usa-alert-heading caution-flag-alert-heading">
-              {headline}
-            </h4>
-          </div>
-          <div>
-            <i
-              style={{
-                float: 'right',
-                margin: '5px',
-              }}
-              className={classNames('fa', {
-                'fa-chevron-up': expanded,
-                'fa-chevron-down': !expanded,
-              })}
-            />
+      <div className="usa-alert-body ">
+        <button
+          className="caution-flag-toggle"
+          onClick={() => toggleExpansion(!expanded)}
+        >
+          <div className="vads-u-display--flex">
+            <div>
+              <h4 className="usa-alert-heading caution-flag-alert-heading">
+                {headline}
+              </h4>
+            </div>
+            <div>
+              <i
```

Comment:
Icon needs `aria-hidden="true"` so it doesn't announce the glyph to screen readers